env/Makefile

```diff
@@ -60,7 +62,7 @@ batchdb-oracle.properties: batchdb-oracle.properties.php ../vars
 .PHONY: $(TRANSMART_BATCH_FILE)
 
 $(GROOVY_ZIP):
-	curl -f "$(GROOVY_URL)" > $@
+	curl -L "$(GROOVY_URL)" > $@
```

Comment:
`-f` should be kept, otherwise a failure downloading the file will result in an empty file and there will be no attempt to download it again.